.gitignore

```diff
@@ -105,3 +105,4 @@ ENV/
 # jwt keys
 keys/*.pem
 tests/resources/keys/*.pem
+1
```

Comment:
what is this?

Reply:
I think this is something vscode added automatically. I can remove it